src/Features/Core/Portable/Completion/ClassificationOptionsStorage.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.CodeAnalysis.Options;
+
+namespace Microsoft.CodeAnalysis.Classification
+{
+    // TODO: Move to EditorFeatures https://github.com/dotnet/roslyn/issues/59184
```

Comment:
doesn't this need to be in features to allow lsp to access classification options?

Reply:
Hmm that seems like the opposite of the original plan - then LSP requires VS dependencies which aren't really needed

The lsp pieces to interface with the lsp client I agree belong in editorfeatures, but for the rest of the server it should just need features (also so it can run OOP)